web/src/main/java/org/akaza/openclinica/control/admin/SpreadSheetTableRepeating.java

```diff
@@ -2810,7 +2810,7 @@ public String stripQuotes(String subj) {
             returnme += subjarray[subjarray.length - 1];
         }
         return returnme;
-    }
+    }*/
```

Comment:
It's better to delete the code that is not being used instead of commenting it out. We can always look at git history to figure out why this method was removed.

Reply:
Ok it's all set.